template-view-duplicates.php

```diff
@@ -13,20 +13,23 @@
 $dt_duplicates = wp_unslash( $dt_contacts->get_all_duplicates() );
 ?>
 
-    <div id="content">
+    <div id="content" class="duplicates-page">
 
         <div id="inner-content" class="grid-x grid-margin-x">
 
             <main id="main" class="large-12 medium-12 cell" role="main">
                 <div class="bordered-box">
-                    <h3><?php esc_html_e( 'Duplicate Contacts', 'zume' ) ?></h3>
+                    <h3><?php esc_html_e( 'Duplicate Contacts', 'zume' ) ?><button class="help-button float-right" data-section="duplicates-template-help-text">
+                        <img class="help-icon" src="<?php echo esc_html( get_template_directory_uri() . '/dt-assets/images/help.svg' ) ?>"/>
+                    </button></h3>
 
+                    <?php //@// TODO: use an if statement to show a message if not duplicates were found. Else show the following table ?>
                     <table id='table-duplicates'>
                         <?php
                         foreach ( $dt_duplicates as $dt_id => $dt_duplicate ) {
                             if ($dt_duplicate['count'] <= 0) { continue; }
                             echo "<form name='merge_".esc_html( $dt_id )."' method='POST' action='".esc_html( site_url() )."/contacts/".esc_html( $dt_id )."'><input type='hidden' name='dt_contact_nonce' value='".esc_attr( wp_create_nonce() )."'/></form><tr id='".esc_html( $dt_id )."'><td><a>".esc_html( $dt_duplicate['name'] )."</a></td><td>
-                            <a>" . esc_html( $dt_duplicate['count'] ). esc_html_e( 'duplicates' ) . "</a></td></tr>";
+                            <a>" . esc_html( $dt_duplicate['count'] ). /* esc_html_e( 'duplicates' ) . */ " duplicates</a></td></tr>";
```

Comment:
The line should be `<a>" . esc_html( $dt_duplicate['count'] ). esc_html__( 'duplicates', 'disciple_tools' ) . "</a></td>`

Reply:
Thanks - I spent way too long trying to try and resolve that.